assets/content/cookbook/Beginner/string-variable-reflection.md

```diff
@@ -0,0 +1,38 @@
+[tags]: / "beginner,other"
+
+# Using string as variable (with reflection)
+
+Extract variable name from string and using it as variable
```

Comment:
Grammar aside, I think we might have to tweak this just a little bit to make it a little clearer. I propose the following:

```markdown
> This snippet shows how to use a string as a variable identifier.
```